testsuite/MDAnalysisTests/core/test_atomgroup.py

```diff
@@ -23,19 +23,16 @@
 
 from glob import glob
 import itertools
-import os
 from os import path
-from unittest import TestCase
 
 import numpy as np
 import warnings
 
 from numpy.testing import (
-    assert_,
-    assert_array_equal,
     assert_almost_equal,
     assert_equal,
-    assert_array_almost_equal,
+    assert_array_equal,
```

Comment:
we're not using `assert_array_equal`

Reply:
@richardjgowers The `dtype` causes a mismatch in this case. How do I work around it?

```
E           AssertionError: 
E           Arrays are not almost equal to 7 decimals
E           
E           (mismatch 33.3333333333%)
E            x: array([ 2.6028271, -1.1962211,  0.       ], dtype=float32)
E            y: array([ 2.6028272, -1.1962211,  0.       ])
```

Reply:
which test causes the dtype issue?

Reply:
in class `TestAtomGroup`
```
    def test_index_slice_empty(self, universe):
        u = universe
        assert_equal(u.atoms[0:0], [],
                           "making an empty AtomGroup failed")
```